doc/modules/preprocessing.rst

```diff
@@ -71,31 +82,10 @@ Scaled data has zero mean and unit variance::
 
 ..    >>> print_options = np.set_printoptions(print_options)
 
-The ``preprocessing`` module further provides a utility class
-:class:`StandardScaler` that implements the ``Transformer`` API to compute
-the mean and standard deviation on a training set so as to be
-able to later reapply the same transformation on the testing set.
-This class is hence suitable for use in the early steps of a
-:class:`~sklearn.pipeline.Pipeline`::
-
-  >>> scaler = preprocessing.StandardScaler().fit(X_train)
-  >>> scaler
-  StandardScaler()
-
-  >>> scaler.mean_
-  array([1. ..., 0. ..., 0.33...])
-
-  >>> scaler.scale_
-  array([0.81..., 0.81..., 1.24...])
-
-  >>> scaler.transform(X_train)
-  array([[ 0.  ..., -1.22...,  1.33...],
-         [ 1.22...,  0.  ..., -0.26...],
-         [-1.22...,  1.22..., -1.06...]])
-
-
-The scaler instance can then be used on new data to transform it the
-same way it did on the training set::
+This class implements the ``Transformer`` API to compute the mean and
+standard deviation on a training set so as to be able to later re-apply the
+same transformation on the testing set. This class is hence suitable for
+use in the early steps of a :class:`~sklearn.pipeline.Pipeline`::
```

Comment:
I decided not to do it here since there are similar examples of how to use the Pipeline class in the corresponding section of the documentation, i.e. to keep this article less convoluted. Though, I can copy and paste it here if you think that will be beneficial.

Reply:
Yes, I think this section should explicitly illustrate the use of a pipeline

Reply:
ok